lib/kubernetes-deploy/runner.rb

```diff
@@ -113,11 +116,15 @@ def run(verify_result: true, allow_protected_ns: false, prune: true)
       end
 
       if verify_result
-        deploy_resources(resources, prune: prune, verify: true)
+        ::StatsD.measure('duration.normal_resources', tags: %W(namespace:#{@namespace} sha:#{ENV['REVISION']})) do
+          deploy_resources(resources, prune: prune, verify: true)
+        end
         record_statuses(resources)
         success = resources.all?(&:deploy_succeeded?)
       else
-        deploy_resources(resources, prune: prune, verify: false)
+        ::StatsD.measure('duration.normal_resources', tags: %W(namespace:#{@namespace} sha:#{ENV['REVISION']})) do
```

Comment:
Yup, I thought about this but decided to leave it out of here since we already have `location` etc tags to make that difference